browser/brave_shields/brave_shields_tab_helper.cc

```diff
@@ -475,6 +482,43 @@ void BraveShieldsTabHelper::HandleItemAllowedOnce(
   }
 }
 
+void BraveShieldsTabHelper::HandleWebcompatFeatureInvoked(
+    ContentSettingsType webcompat_content_settings) {
+  if (webcompat_content_settings > ContentSettingsType::BRAVE_WEBCOMPAT_NONE &&
+      webcompat_content_settings < ContentSettingsType::BRAVE_WEBCOMPAT_ALL) {
+    webcompat_features_invoked_.insert(webcompat_content_settings);
+  }
+
+  for (Observer& obs : observer_list_) {
+    obs.OnResourcesChanged();
+  }
+}
+
+void BraveShieldsTabHelper::SetWebcompat(
+    ContentSettingsType webcompat_settings_type,
+    bool disabled) {
+  ControlType control_type = disabled ? ControlType::ALLOW : ControlType::BLOCK;
```

Comment:
this is unclear. Pls consider renaming this function to `SetWebcompatEnabled` and renaming the parameter to `enabled`.

sidenote: why `ASK` is a even a valid value (and is a default one)? why not use only `ALLOW/BLOCK`?

Reply:
fixed, and `CONTENT_SETTING_ASK` has been removed